firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -47,6 +47,7 @@
   private UpdateTaskImpl cachedUpdateAppTask;
   private AppDistributionReleaseInternal cachedLatestRelease;
   private final SignInStorage signInStorage;
+  private boolean basicConfiguration = false;
```

Comment:
Does this need to be a class variable? I'm worried about how we always need to remember to set this back to false. It feels like it could be error prone.

Could we just overload updateApp so that internally, we call it with `showProgressInNotificationManager=true`, while the public method calls it with 

so we'd have two updateApps:

private updateApp(boolean ....) {
  // same as now
} 

public update app(){
  updateApp(false)
}